From ed2ea5e944aae002286491ab98d1419c97b4f343 Mon Sep 17 00:00:00 2001
From: Yi Zhao <yi.zhao@windriver.com>
Date: Mon, 17 Sep 2018 08:34:05 +0800
Subject: [PATCH] wr-policy: add path for mke2fs.* and tune2fs.*

The mke2fs and tune2fs commands are switched to use update-alternatives
in oe-core commit 81dc858a24cc5b5dc547356eb22f00dde9801b6f. We need to
add policy for mke2fs.* and tune2fs.*

Upstream-Status: Pending

Signed-off-by: Yi Zhao <yi.zhao@windriver.com>
---
 policy/modules/system/fstools.fc | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/policy/modules/system/fstools.fc b/policy/modules/system/fstools.fc
index b45f550..9842446 100644
--- a/policy/modules/system/fstools.fc
+++ b/policy/modules/system/fstools.fc
@@ -23,6 +23,7 @@
 /sbin/make_reiser4	--	gen_context(system_u:object_r:fsadm_exec_t,s0)
 /sbin/mkdosfs		--	gen_context(system_u:object_r:fsadm_exec_t,s0)
 /sbin/mke2fs		--	gen_context(system_u:object_r:fsadm_exec_t,s0)
+/sbin/mke2fs.*		--	gen_context(system_u:object_r:fsadm_exec_t,s0)
 /sbin/mke4fs		--	gen_context(system_u:object_r:fsadm_exec_t,s0)
 /sbin/mkfs.*		--	gen_context(system_u:object_r:fsadm_exec_t,s0)
 /sbin/mkraid		--	gen_context(system_u:object_r:fsadm_exec_t,s0)
@@ -42,6 +43,7 @@
 /sbin/swapoff\.util-linux		--	gen_context(system_u:object_r:fsadm_exec_t,s0)
 /sbin/swapon.*		--	gen_context(system_u:object_r:fsadm_exec_t,s0)
 /sbin/tune2fs		--	gen_context(system_u:object_r:fsadm_exec_t,s0)
+/sbin/tune2fs.*		--	gen_context(system_u:object_r:fsadm_exec_t,s0)
 /sbin/zdb		--	gen_context(system_u:object_r:fsadm_exec_t,s0)
 /sbin/zhack		--	gen_context(system_u:object_r:fsadm_exec_t,s0)
 /sbin/zinject		--	gen_context(system_u:object_r:fsadm_exec_t,s0)
@@ -81,6 +83,7 @@
 /usr/sbin/make_reiser4		--	gen_context(system_u:object_r:fsadm_exec_t,s0)
 /usr/sbin/mkdosfs		--	gen_context(system_u:object_r:fsadm_exec_t,s0)
 /usr/sbin/mke2fs		--	gen_context(system_u:object_r:fsadm_exec_t,s0)
+/usr/sbin/mke2fs.*		--	gen_context(system_u:object_r:fsadm_exec_t,s0)
 /usr/sbin/mke4fs		--	gen_context(system_u:object_r:fsadm_exec_t,s0)
 /usr/sbin/mkfs.*		--	gen_context(system_u:object_r:fsadm_exec_t,s0)
 /usr/sbin/mkraid		--	gen_context(system_u:object_r:fsadm_exec_t,s0)
@@ -100,6 +103,7 @@
 /usr/sbin/swapoff		--	gen_context(system_u:object_r:fsadm_exec_t,s0)
 /usr/sbin/swapon.*		--	gen_context(system_u:object_r:fsadm_exec_t,s0)
 /usr/sbin/tune2fs		--	gen_context(system_u:object_r:fsadm_exec_t,s0)
+/usr/sbin/tune2fs.*		--	gen_context(system_u:object_r:fsadm_exec_t,s0)
 /usr/sbin/zdb			--	gen_context(system_u:object_r:fsadm_exec_t,s0)
 /usr/sbin/zhack			--	gen_context(system_u:object_r:fsadm_exec_t,s0)
 /usr/sbin/zinject		--	gen_context(system_u:object_r:fsadm_exec_t,s0)
-- 
2.13.3

